dev-docs/source/Testing/MuonAnalysis_test_guides/Muon_EMU.rst

```diff
@@ -77,6 +75,9 @@ Superconducting Copper Test
 
 **Time required 5 - 10 minutes**
 
+To use the **Model Fitting** tab see the :ref:`Muon_Feature_Flags-ref` documentation. You will
```

Comment:
Here the text links to the ``Muon Feature Flags`` section of the interface. Since this is a manual testing for developers, it may happen that they are testing on a nightly or on a local build of mantid. In that case, shouldn't it be better to remind here that the path to the `Mantid.User.Properties` file will be different in those two cases (at least for windows). It may save time for the developers testing it, if they don't realize that `bin` folder is different from nightly or local build.

Reply:
I think it should be ok to leave the path just as the `MantidInstall` path. When doing manual testing/smoke testing, developers should make sure they are using a nightly and not a local build. This is because we want the testing to be done in an environment which is as close to the release candidate as possible. If we do testing using a local build, there may be something that works that wouldn't work for a proper Mantid install (e.g. the Indirect interface category only went missing for nightly installs and not local builds #36645)